packages/webapp/src/routes/land/check-ownership-proof-file.js

```diff
@@ -17,14 +18,22 @@ const handlers = {
     const checkLandOwnership = request.payload.checkLandOwnership
     const context = getContext(request)
     request.yar.set(constants.redisKeys.LAND_OWNERSHIP_CHECKED, checkLandOwnership)
+    let ownershipProofs = request.yar.get(constants.redisKeys.LAND_OWNERSHIP_PROOFS) || []
+
     if (checkLandOwnership === 'no') {
       await deleteBlobFromContainers(context.fileLocation)
       request.yar.clear(constants.redisKeys.LAND_OWNERSHIP_LOCATION)
+      _.remove(ownershipProofs, (item) => item === path.basename(context.fileLocation))
```

Comment:
Not sure why you are doing this (remove operation followed by setting the redis key) as the file won't have been added to `LAND_OWNERSHIP_PROOFS` at this stage.

Also, even though I don't think you need this line of code, I wouldn't use lodash here, it's an unnecessary dependency for such as simple operation that can be achieved with the built in Array methods (filter or splice with findIndex)